packages/mds-cache/types.ts

```diff
@@ -37,16 +37,16 @@ export type StringifiedTripEvent = Stringify<Omit<Omit<TripEvent, 'annotation'>,
   annotation: Stringify<AnnotationData> | null
   gps: Stringify<GpsData> | null
 }
-export type StringifiedTripsEvents = { [trip_id: string]: StringifiedTripEvent[] }
-export type StringifiedAllTripsEvents = { [vehicle_id: string]: StringifiedTripsEvents }
+export type StringifiedTripEvents = string
```

Comment:
All of the typedefs where the `Stringified` versions are set to `string` could definitely be better -- take a look at `StringifiedEvent`

Reply:
So this ones a but tricky. I'm familiar with the structuring of these, but for this specific response it's a string of a list of unknown length of StrigifiedTripEvents. Open to ideas for a workaround, but for now this is functional.

Reply:
If you have a type for TripEvent (which we should have but I don't think we do currently), then you can do `export type StringifiedTripEvents = Stringify<TripEvents>`

Reply:
Ah ok, I have StringifiedTripEvent for that

Reply:
Now I remember why I didn't use that approach, Stringify<StringifiedTripEvent[]> results in a string[] type not a string.